db/view/view_update_generator.cc

```diff
@@ -39,10 +39,22 @@ future<> view_update_generator::start() {
                 if (result == stop_iteration::yes) {
                     break;
                 }
-                entry.t->move_sstable_from_staging_in_thread(entry.sst);
-                _registration_sem.signal();
+                auto t = entry.t.get();
```

Comment:
This change is also not explained.

Reply:
right about `_registration_sem.signal()`
It was dropped inadvertently.

Reply:
fixed